crates/uv-resolver/src/lock/requirements_txt.rs

```diff
@@ -41,13 +44,18 @@ impl<'lock> RequirementsTxtExport<'lock> {
         install_options: &'lock InstallOptions,
     ) -> Result<Self, LockError> {
         let size_guess = target.lock().packages.len();
-        let mut petgraph = Graph::with_capacity(size_guess, size_guess);
+        let mut graph = Graph::<Node<'lock>, Edge<'lock>>::with_capacity(size_guess, size_guess);
         let mut inverse = FxHashMap::with_capacity_and_hasher(size_guess, FxBuildHasher);
 
         let mut queue: VecDeque<(&Package, Option<&ExtraName>)> = VecDeque::new();
         let mut seen = FxHashSet::default();
+        let mut conflicts = if target.lock().conflicts.is_empty() {
+            None
+        } else {
+            Some(FxHashMap::default())
```

Comment:
It looks like `MarkerTree::TRUE` is the only thing that's ever stored as a value in this particular map? Or am I wrong about that? If so, I think you could use a `FxHashSet` here instead.